packages/node-sdk/src/index.ts

```diff
@@ -0,0 +1,10 @@
+export type { Client } from "./client";
```

Comment:
Are we only exporting the type here? Seems like we should export the actual client.

I think we should name it `BucketClient` in the public interface. `Client` is quite overloaded and might result in forcing the consumer to change the import names